Makefile

```diff
@@ -8,16 +8,7 @@ DOCKER_WORKSPACE := "/markdown"
 MOUNTS = --volume ${PWD}:${DOCKER_WORKSPACE} \
 	--volume ${DOCKER_WORKSPACE}/node_modules
 
-ifeq ($(USE_LEGACY), true)
-dockerfile = -f Dockerfile.legacy
-endif
-
-ifeq ($(USE_LEGACY), true)
-dockerfile = -f Dockerfile.legacy
-endif
-
```

Comment:
The 'legacy' Dockerfile was for supporting node v18. node 18 is very close to end of life and we weren't even running tests against it anymore.